bibliopixel/layout/layout.py

```diff
@@ -12,19 +12,22 @@ def construct(cls, project, **desc):
         """Construct a layout."""
         return cls(project.drivers, maker=project.maker, **desc)
 
-    def __init__(self, drivers, threadedUpdate, brightness,
-                 maker=data_maker.MAKER, **kwds):
+    def __init__(self, drivers, threadedUpdate=False, brightness=255,
+                 maker=data_maker.MAKER, color_list=None, **kwds):
```

Comment:
Can we get some docstrings here so we know what color_list is?? As we are going to be finalizing all the docs soon we might as well start adding stuff as the code is touched.

Reply:
Done!